sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -1442,10 +1436,9 @@ def write(self, row):
     try:
       self._avro_writer.write(row)
     except (TypeError, ValueError) as ex:
-      raise_with_traceback(
-          ex.__class__(
-              "Error writing row to Avro: {}\nSchema: {}\nRow: {}".format(
-                  ex, self._avro_writer.schema, row)))
+      ex.__class__(
```

Comment:
you have to `raise` this, right?

Reply:
Exactly right!
And I found that I've typed `with_trackback` in [l.1441](https://github.com/apache/beam/pull/14388/files/91417779134e69c9871ec8f69bbbe1115d9d491b#diff-5427a5d3887eb695cefde082c58575a2372972996b547d55961abeb4f7bc3debR1441) by mistake, `with_traceback` is correct.
Both are fixd at 3c71066